ci_framework/hooks/playbooks/validate_podified_deployment.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Make sure Nova api service is up and running
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        oc get pods --selector service=nova-api -o jsonpath={.items[*].status.phase}
+      register: nova_service
+      until: nova_service.stdout == "Running"
+      changed_when: false
+      retries: 100
+      delay: 50
+
+    - name: List compute and network resources
+      when: podified_validation | default ('false') | bool
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        oc project openstack;
+        oc rsh openstackclient openstack compute service list;
+        oc rsh openstackclient openstack network agent list;
```

Comment:
This would change the active namespace on the OCP cluster, wouldn't a better practice be to use the namespace parameter in the `oc` command itself?